index.bs

```diff
@@ -531,33 +515,86 @@ The {{ML/createContext()}} method steps are:
 1. Switch on the method's first argument:
     <dl class=switch>
     <dt>{{MLContextOptions}}
-    <dd>Set |context|'s [=context type=] to [=default-context|default=].
+    <dd>Set |context|.{{[[contextType]]}} to [=default-context|default=].
+    <dd>Set |context|.{{[[devicePreference]]}} to the value of {{MLContextOptions}}'s {{devicePreference}} member.
+    <dd>Set |context|.{{[[powerPreference]]}} to the value of {{MLContextOptions}}'s {{powerPreference}} member.
```

Comment:
Is the double square brackets here intentional? Can we remove them? It looks unpolished and makes the text harder to read in the final rendering.

Reply:
I fully agree it does not look pretty, and I'm not a fan of this look personally, but this is the convention so I suggest we align with it. That is, it is intentional.

(Web IDL inherits the square brackets convention for internal slots from ECMA-262: https://webidl.spec.whatwg.org/#ecmascript-binding)